src/components/Header/Header.tsx

```diff
@@ -0,0 +1,12 @@
+import { useState } from "react"
+import { fetchHeader } from "../../utils/strapiAPIs"
+
+export default function Header() {
+  const [title, setTitle] = useState<string>(fetchHeader())
+
+  return (
+    <div className='header-container p-5 bg-darkPrimary lg:w-full'>
+      <h1 className='text-textPrimary opacity-50 lg:text-8xl'>{title}</h1>
```

Comment:
Same change as above, change it to `text-light-500` instead of `text-textPrimary`